scripts/build_package.py

```diff
@@ -166,12 +166,15 @@ def handle_external_file(dep, clean, env, verbose):
     download_cmd.append(file_url)
 
     option = source_package['option']
-    download_cmd.append(option)
+    download_cmd.extend(option)
```

Comment:
@gyorb Do not we need to handle the case when `option` is not set? Every source package should set this?